assert/assertions.go

```diff
@@ -150,6 +150,8 @@ func copyExportedFields(expected interface{}) interface{} {
 // structures.
 //
 // This function does no assertion of any kind.
+//
+// Deprecated: Use EqualExportedValues instead
```

Comment:
End the sentence.

```suggestion
// Deprecated: Use EqualExportedValues instead.
```

Reply:
Thanks! How does the godoc linking work here?